src/siwe-client.ts

```diff
@@ -76,17 +74,7 @@ export class SiweImpl implements SiweApiClient {
       throw new Error(`Received error response on login: ${responseText}`)
     }
 
-    const headerSetCookie = (response.headers as any).raw()[
-      'set-cookie'
-    ] as Array<string>
-
-    if (headerSetCookie) {
-      await Promise.all(
-        headerSetCookie.map(async (cookie: string) => {
-          await this.cookieJar.setCookie(cookie, this.config.loginUrl)
-        }),
-      )
-    }
+    this._loginHeader = response.headers
```

Comment:
can we update tests to check if this field is set?

Reply:
Will do